docs/blog/posts/announcing-gemini-tool-calling-support.md

```diff
@@ -0,0 +1,116 @@
+---
+draft: False
+date: 2024-09-03
+authors:
+  - ivanleomk
+---
+
+# Announcing Gemini and VertexAI Tool Calling Support
+
+We're excited to announce that `instructor` now supports tool calling for both the Gemini SDK and the VertexAI SDK.
+
+A special shoutout to [Sonal](https://x.com/sonalsaldanha) for his contributions to the Gemini Tool Calling support.
+
+Let's walk through a simple example of how to use these new features
+
+## Installation
+
+To get started, install the latest version of `instructor`. Depending on whether you're using Gemini or VertexAI, you should install the following:
+
+=== "Gemini"
+
+    ```bash
+    pip install "instructor[gemini]"
+    ```
+
+=== "VertexAI"
+
+    ```bash
+    pip install "instructor[vertexai]"
+    ```
+
+This ensures that you have the necessary dependencies to use the Gemini or VertexAI SDKs with instructor.
+
+## Getting Started
```

Comment:
make a recommendation on which one to use

Reply:
Recommendation made

We recommend using the Gemini SDK over the VertexAI SDK for two main reasons.

1. Compared to the VertexAI SDK, the Gemini SDK comes with a free daily quota of 1.5 billion tokens to use for developers.
2. The Gemini SDK is significantly easier to setup, all you need is a `GOOGLE_API_KEY` that you can generate in your GCP console. THe VertexAI SDK on the other hand requires a credentials.json file or an OAuth integration to use.